vllm/attention/layer.py

```diff
@@ -190,7 +190,7 @@ def __init__(
         # opaque custom op. For other platforms, we directly call them
         # and let torch.compile handle them.
         self.use_direct_call = not current_platform.is_cuda_alike(
-        ) and not current_platform.is_cpu()
+        ) and not current_platform.is_cpu() and not current_platform.is_xpu()
```

Comment:
Could you make this a new property on the platform interface, called `opaque_attention_op`?

Reply:
sure, added.